flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -293,7 +294,8 @@ public StreamExecutionEnvironment(
         this.configure(this.configuration, this.userClassloader);
     }
 
-    protected ClassLoader getUserClassloader() {
+    @VisibleForTesting
+    public ClassLoader getUserClassloader() {
```

Comment:
We should not make this method public because `StreamExecutionEnvironment` is `@Public`, especially given that this change is for test only.

Reply:
What's your concern about exposing this method for tests? Can you give me some advice about what classloader we should use?  `env.getClass().getClassLoader()`?

Reply:
When it is public, users will also have access to it, which is what we want. 
Will these test pass if we pass the default classloader in as the user classloader?

Reply:
The ci of runtime has passed.